lesson2 таймеры

vika-Kov opened the pull request.

vika-Kov committed:
lesson2 таймеры

Om1X reviewed (changes requested):
задание не работает должным образом. Всегда вижу отсчет таймеров от 3 до 0 и все.

vika-Kov committed:
Update index.js

const seconds изначально содержал тестовое значение - 3
Раскомментировала правильный расчет таймеров
